### ISSUE TITLE
[Popover]: [Anything which comes under the Label component is not read by the screen reader also the focus is not set to the Label message.]

### ISSUE THREAD

**Issue by abhi091000** [SAP/ui5-webcomponents-react#5593]
### Describe the bug

<>
  <Button
    id="openPopoverBtn"
    onClick={function _a(){}}
  >
    Open Popover
  </Button>
  <Popover
    className="footerPartNoPadding"
    headerText="Popover Header"
    horizontalAlign="Center"
    onAfterClose={function _a(){}}
    onAfterOpen={function _a(){}}
    onBeforeClose={function _a(){}}
    onBeforeOpen={function _a(){}}
    opener="openPopoverBtn"
    placementType="Right"
    verticalAlign="Center"
  >
    <Label>
      Press "Escape" or click outside to close the Popover
    </Label>
  </Popover>
</>
we also have the similar type of code in one of our component, but by deafault the focus is not being set on the Label and cannot read the message inside it.

### Reproduction steps

https://jira.eurekacloud.io/browse/CRP-91674

### UI5 Web Components for React Version

0.27.5

### UI5 Web Components Version

1.6.0

### Browser

Chrome

### Operating System

window 11

### Additional Context

https://jira.eurekacloud.io/browse/CRP-91674

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @abhi091000 

please add an isolated example, preferably by using this [StackBlitz](https://stackblitz.com/github/SAP/ui5-webcomponents-react/tree/main/templates/vite-ts?file=src%2FApp.tsx) template.
